docs/source/releases/v2.0.rst

```diff
@@ -132,6 +132,13 @@ Minor changes
 - Added a ``datetime_filters`` tag library that provides a ``timedelta`` template
   filter for rendering time deltas in human readable format.
 
+- Redirection to parent product detail page was disabled for variant products,
```

Comment:
I think this needs to move to the "backward incompatible changes" section, rather than "minor changes", because it can break behaviour for existing projects.

Reply:
Indeed.